From 720bf06e82b3ef5156d3840dc4c0d3d691ec5a9d Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 28 Feb 2024 11:05:21 +0000
Subject: [PATCH 0913/1085] dtoverlays: rotate override needs to update
 "rotation" for drm mode

Whilst the fbtft driver uses the DT property "rotate" to set the display
rotation, the tinydrm ili9486 driver uses "rotation". The overlay was
only updating "rotate" from the override, so add in "rotation".

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/piscreen-overlay.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/piscreen-overlay.dts
+++ b/arch/arm/boot/dts/overlays/piscreen-overlay.dts
@@ -96,7 +96,8 @@
 	};
 	__overrides__ {
 		speed =		<&piscreen>,"spi-max-frequency:0";
-		rotate =	<&piscreen>,"rotate:0";
+		rotate =	<&piscreen>,"rotate:0",
+				<&piscreen>,"rotation:0";
 		fps =		<&piscreen>,"fps:0";
 		debug =		<&piscreen>,"debug:0";
 		xohms =		<&piscreen_ts>,"ti,x-plate-ohms;0";
